CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.3.14-M1
```

Comment:
Release 0.3.13-M1 is not released yet. 
0.3.14-M1 is not needed in the changelog. Please use 0.3.13-M1